### PR TITLE
add documentation on withAtLeastArguments and withAtLeastIdenticalArguments methods

### PR DESCRIPTION
**Pull request by agallou**

### CODE REVIEW

File: source/en/asserters/mock.inc.rst [hunk 1]
```diff
@@ -210,6 +210,58 @@ withIdenticalArguments
...
+      ->and($mock->test('a', 'b'))
+      ->mock($mock)
+      ->call('test')
+            ->withAtLeastArguments(array('a'))->once() //passes
+            ->withAtLeastArguments(array('a', 'b'))->once() //passes
+            ->withAtLeastArguments(array('c'))->once() //fails
+   ;
+
+.. warning::
+| ``withAtLeastArguments`` does not check the arguments type.
```
Comment (marmotz) [inline source/en/asserters/mock.inc.rst]: missing indent ?

File: source/en/asserters/mock.inc.rst [hunk 2]
```diff
@@ -210,6 +210,58 @@ withIdenticalArguments
...
+       ->and($mock->test(1, 2))
+       ->mock($mock)
+           ->call('test')
+           ->withAtLeastIdenticalArguments(array(1))->once() //passes
+           ->withAtLeastIdenticalArguments(array(1, 2))->once() //passes
+           ->withAtLeastIdenticalArguments(array('1'))->once() //fails
+   ;
+
+.. warning::
+| ``withAtLeastIdenticalArguments`` checks the arguments type.
```
Comment (marmotz) [inline source/en/asserters/mock.inc.rst]: missing indent ?

File: source/fr/assertions/mock.inc.rst [hunk 3]
```diff
@@ -210,6 +210,58 @@ withIdenticalArguments
...
+      ->and($mock->test('a', 'b'))
+      ->mock($mock)
+      ->call('test')
+            ->withAtLeastArguments(array('a'))->once() //passes
+            ->withAtLeastArguments(array('a', 'b'))->once() //passes
+            ->withAtLeastArguments(array('c'))->once() //fails
+   ;
+
+.. warning::
+| ``withAtLeastArguments`` ne teste pas le type des arguments.
```
Comment (marmotz) [inline source/fr/assertions/mock.inc.rst]: missing indent ?

File: source/fr/assertions/mock.inc.rst [hunk 4]
```diff
@@ -210,6 +210,58 @@ withIdenticalArguments
...
+       ->and($mock->test(1, 2))
+       ->mock($mock)
+           ->call('test')
+           ->withAtLeastIdenticalArguments(array(1))->once() //passes
+           ->withAtLeastIdenticalArguments(array(1, 2))->once() //passes
+           ->withAtLeastIdenticalArguments(array('1'))->once() //fails
+   ;
+
+.. warning::
+| ``withAtLeastIdenticalArguments`` teste le type des arguments.
```
Comment (marmotz) [inline source/fr/assertions/mock.inc.rst]: missing indent ?

File: source/fr/assertions/mock.inc.rst [hunk 5]
```diff
@@ -210,6 +210,58 @@ withIdenticalArguments
...
+   | Si vous souhaitez vérifier également leurs types, utilisez :ref:`withAtLeastIdenticalArguments <with-at-least-identical-arguments>`.
+
+
+
+.. _with-at-least-identical-arguments:
+
+withAtLeastIdenticalArguments
+`````````````````````````````
+
+``withAtLeastIdenticalArguments`` permet de spécifier les paramètres minimum attendus lors de l'appel à la méthode testée (voir :ref:`call <call-anchor>`) du mock testé.
```
Comment (marmotz) [inline source/fr/assertions/mock.inc.rst]: minimumS

**Comment by agallou** [comment]
@marmotz indentation and typo fixed

**Comment by marmotz** [comment]
Thanks !

**Pull request merged**